modules/post/windows/escalate/ms10_073_kbdlayout.rb

```diff
@@ -85,7 +85,7 @@ def run
 
     ring0_code =
       #"\xcc" +
-      # save registers -- necessary for successfuly recovery
+      # save registers -- necessary for successfully recovery
```

Comment:
Successful.

Reply:
Not all search and replace goes as planned 😞. Changing!